LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWDLCHelpers.uc

```diff
@@ -137,6 +137,21 @@ static function bool IsLostTowersNarrativeEnabled()
 	return false;
 }
 
+// Replaces X2StrategyElement_DLC_Day90Techs.static.IsMechanizedWarfareAvailable()
+static function bool IsMechanizedWarfareAvailable()
+{
+    // We need MechanizedWarfare to be available in Integrated DLC mode. This function replaces the
```

Comment:
I know it's a trivial thing, but this is going to drive me nuts. A tab here please.